.github/workflows/docs.yaml

```
@@ -84,15 +85,67 @@ jobs:
           # TODO: enable before going live.
           #env:
           #  RUN_LINK_CHECK: true
-          pnpm astro build \
-            --site "${{ steps.pages.outputs.origin }}" \
-            --base "${{ steps.pages.outputs.base_path }}"
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            pnpm astro build \
+              --site "https://tenzir-pr-${{ github.event.pull_request.number }}.surge.sh"
```

```suggestion
              --site "https://tenzir-docs-${{ github.event.pull_request.number }}.surge.sh"
```

How about this slug?